cmd/fleetctl/preview.go

```diff
@@ -673,6 +676,26 @@ func downloadOrbitAndStart(destDir, enrollSecret, address, orbitChannel, osquery
 	return nil
 }
 
+// cleanUpSocketFiles cleans up fleet-osqueryd's socket file
+// ("orbit-osquery.em") and osquery extension socket files
+// ("orbit-osquery.em.*").
+func cleanUpSocketFiles(path string) error {
+	entries, err := os.ReadDir(path)
+	if err != nil {
+		return fmt.Errorf("read dir: %w", err)
+	}
+	for _, entry := range entries {
+		if !strings.HasPrefix(entry.Name(), "orbit-osquery.em") {
+			continue
+		}
+		entryPath := filepath.Join(path, entry.Name())
+		if err := os.RemoveAll(entryPath); err != nil {
```

Comment:
Curious why `RemoveAll` is used here instead of `os.Remove`? The comments make it seem like those are files, can they also be directories? If so I'd adjust the comments to reflect that, otherwise I think it should use `os.Remove`.